FEXCore/Source/Interface/Core/Dispatcher/Dispatcher.cpp

```diff
@@ -106,6 +106,30 @@ void Dispatcher::EmitDispatcher() {
     ptrue(ARMEmitter::SubRegSize::i8Bit, PRED_TMP_16B, ARMEmitter::PredicatePattern::SVE_VL16);
```

Comment:
This would seem redundant in the SVE256 case no?

Reply:
Oh I was just matching fill here, will commonise the afp+sve SRA filling and fix there

Reply:
these regs are different :)